prototypes/rohit_magdum/placeholder.txt

```diff
@@ -1 +1,2 @@
 These are changes for pull request. Just to test the review system.
```

Comment:
This should be removed. However i'm fine with keeping it as is since its nonfunctional change